src/docs/style/themes.mdx

```diff
@@ -1,11 +1,23 @@
-import { ImageGrid, DemoCard } from '../../app/components';
+import { ImageGrid, DemoCard, MaterialDesignDescription } from '../../app/components';
 import LightThemeImage from '../../app/assets/design-patterns/theme/theme-light.png';
 import DarkThemeImage from '../../app/assets/design-patterns/theme/theme-dark.png';
 
 # Power Xpert Blue Themes
 
 One of the key benefits of PX Blue is consistency of look & feel between different applications. This is achieved primarily through the use of PX Blue themes.
 
+<MaterialDesignDescription
+    title={'Material Theming Overview'}
+    description={'Material Design\'s brief introduction to theming.'}
```

Comment:
prefer template strings over escape chars...the linter should be throwing complaints for this:
```
`Material Design's brief introduction to theming.`
```